updatecli/updatecli.d/plugin-site-api

```diff
@@ -0,0 +1,53 @@
+name: Bump `plugin-site-api` docker image and `plugin-site` helm chart versions
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  latestRelease:
+    kind: githubrelease
+    name: "Get latest jenkins-infra/plugin-site-api latest version"
+    spec:
+      owner: "jenkins-infra"
+      repository: "docker-plugin-site-issues"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
```

Comment:
We haven't used (git) tags or (GitHub) releases since 2021 for this repository: https://github.com/jenkins-infra/plugin-site-api/tags